src/help/technical.md

```diff
@@ -0,0 +1,140 @@
+#### December 7, 2016
```

Comment:
Why does this end before the history one if this is current updates? Shouldn't this be history?